base/gfspconfig/config.go

```diff
@@ -180,12 +180,13 @@ type ParallelConfig struct {
 	GlobalGcObjectSafeBlockDistance   uint64 `comment:"optional"`
 	GlobalSyncConsensusInfoInterval   uint64 `comment:"optional"`
 
-	UploadObjectParallelPerNode         int   `comment:"optional"`
-	ReceivePieceParallelPerNode         int   `comment:"optional"`
-	DownloadObjectParallelPerNode       int   `comment:"optional"`
-	ChallengePieceParallelPerNode       int   `comment:"optional"`
-	AskReplicateApprovalParallelPerNode int   `comment:"optional"`
-	QuerySPParallelPerNode              int64 `comment:"optional"`
+	SPMigrateBucketParallel             uint64 `comment:"optional"`
```

Comment:
Please note dev-ops team to change the config.

Reply:
will use the exisiting one `GlobalMigrateGVGParallel` instead.